refs/wicg.json

```diff
@@ -47,14 +47,6 @@
         "source": "https://wicg.github.io/admin/biblio.json",
         "repository": "https://github.com/wicg/cors-rfc1918"
     },
-    "CSS-ANIMATION-WORKLET": {
```

Comment:
Where did that one go?

Reply:
The Houdini folks took it. It's in the w3c.json file now.

Reply:
So you're telling me it disappeared, right? 🐇✨🎩